sql/sql_optimizer.h

```diff
@@ -55,6 +55,15 @@ typedef struct st_rollup
   List<Item> *fields;
 } ROLLUP;
 
+enum class OptimizePhase // for Tianmu to indicate which optimization phase.
```

Comment:
refer to googel style: https://google.github.io/styleguide/cppguide.html#Enumerator_Names

Reply:
ACK